src/features/bakery/Bakery.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Modal } from "react-bootstrap";
+
+import bakery from "assets/buildings/bakery.png";
+import soup from "assets/icons/bakery.png";
+
+import { Crafting } from "./components/Crafting";
+import { Action } from "components/ui/Action";
+import { GRID_WIDTH_PX } from "features/game/lib/constants";
+
+export const Bakery: React.FC = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+  return (
+    <div
+      className="z-10 absolute"
+      // TODO some sort of coordinate system
+      style={{
+        width: `${GRID_WIDTH_PX * 100}px`,
+        height: `${GRID_WIDTH_PX * 5.2}px`,
+        left: `calc(50% - ${GRID_WIDTH_PX * -12}px)`,
+        top: `calc(50% - ${GRID_WIDTH_PX * 19}px)`,
+      }}
+    >
+      <img
+        src={bakery}
+        alt="market"
+        onClick={() => setIsOpen(true)}
+        className="cursor-pointer hover:img-highlight"
+        style={{
+          width: "300px",
+          height: "300px",
+        }}
+      />
+      <Action
```

Comment:
<img width="138" alt="Screen Shot 2022-01-25 at 9 28 54 am" src="https://user-images.githubusercontent.com/11745561/150977754-2d7001b6-535f-41ca-9b13-7f081de28f56.png">

Consider using a different food item here. The contrast with the soup and the grey background looks a bit off.

I would suggest finding one that has a dark border/outline